### PR TITLE
Resolve bug causing Anki installation failure

### PR DESCRIPTION
**Pull request by huiyu-turbo**
`rmdir c:\Workspace/S /Q` in the old script raises "The process cannot access the file because it is being used by another process." And this might cause the installation of Anki failure.
Changed this line to remove the specific files inside the folder, not folder itself.

### CODE REVIEW

File: anki/turbo.me [hunk 1]
```diff
@@ -69,7 +69,8 @@ enable ChromiumSupport
 ###################################
 
 batch
-  rmdir c:\Workspace /S /Q
+  del c:\Workspace\Setup.exe /Q
+  del c:\Workspace\GetVersion.ps1 /Q
```
Comment (johnsu) [inline anki/turbo.me:73]: Should be able to do a recursive delete using a flag

**Review by johnsu — CHANGES_REQUESTED** [review]

File: anki/turbo.me (hunk 1 above)
Comment (huiyu-turbo) [reply anki/turbo.me:73]: `rmdir /S /Q` is equivalent to `rm -rf`. I searched online and there's no other flag.

**Comment by johnsu** [comment]
Mine is now failing at the file download:
```
c:\Users\john\s\server>turbo build https://raw.githubusercontent.com/turboapps/turbome/285dbdad6b32816f8f21e08eab5639357988deba/anki/turbo.me
Using VM 23.6.22 from local
Step 0: meta title="Anki"
Step 1: meta namespace="anki"
Step 2: meta name="anki"
Step 3: using gnu/wget
Using image clean:40 from local
Using image wget:1.11.4 from local
Step 4: cmd (mkdir, c:\Workspace)

Step 5: workdir c:\Workspace
Step 6: batch
        cmd ("echo $webpage = Invoke-WebRequest -Uri "https://apps.ankiweb.net/downloads/archive/" >> GetVersion.ps1")
        cmd ("echo $releases = $webpage.Links.href ^| where {$_ -like "*windows-qt6.exe"} ^| sort >> GetVersion.ps1")
        cmd (echo $latest = $releases[-1] >> GetVersion.ps1)
        cmd ("echo $version = [regex]::Match($latest, "\d+\.\d+\.\d+").Value >> GetVersion.ps1")
        cmd (echo $version >> GetVersion.ps1)

Step 7: cmd ("powershell -executionpolicy remotesigned -File "GetVersion.ps1"")

2.1.65
Step 8: cmd (echo, https://github.com/ankitects/anki/releases/download/2.1.65/anki-2.1.65-windows-qt6.exe)

https://github.com/ankitects/anki/releases/download/2.1.65/anki-2.1.65-windows-qt6.exe
Step 9: cmd (wget, --no-check-certificate, --no-verbose, -O, Setup.exe, %url%)

SYSTEM_WGETRC = c:/progra~1/wget/etc/wgetrc
syswgetrc = c:/progra~1/wget/etc/wgetrc
Step 10: cmd (Setup.exe, /S, /D=c:\Anki)

Step 11: enable ChromiumSupport
Step 12: batch
        cmd (del c:\Workspace\Setup.exe /Q)
        cmd (del c:\Workspace\GetVersion.ps1 /Q)
        cmd (rmdir c:\wget /S /Q)

Could Not Find c:\Workspace\Setup.exe
Step 13: meta tag="2.1.65"
Step 14: startup file (c:\Anki\anki.exe
```

**Comment by huiyu-turbo** [comment]
I remember previously you also had a similar issue and wget crashed in the container. Did the crash happen again?

**Comment by johnsu** [comment]
> I remember previously you also had a similar issue and wget crashed in the container. Did the crash happen again?

Do you think [Joshua's](https://github.com/turboapps/turbome/commit/a323b2c71b1808e19541693ab122382af636aef3) powershell usage of wget will work better? I had tried that and it worked on my other computer.

**Comment by johnsu** [comment]
Please open a bug on the items we discussed to improve the robustness of test data and answer verifications.

**Comment by johnsu** [comment]
Yea if its 1 less component that needs to be imported (+has some nice output ui) I'd prefer we start using it more. Seems strange it would be slower.

**Comment by huiyu-turbo** [comment]
Revised the script, taking inspiration from OBS Turbo Script. Everything looks good on my computer. Please try it.
The only issue is that `wget` is 10x slower.

**Comment by johnsu** [comment]
I tried it out and yea its ridiculously slow - worth searching internet for an answer.

**Comment by huiyu-turbo** [comment]
The cause is the progress bar: https://stackoverflow.com/questions/28682642/powershell-why-is-using-invoke-webrequest-much-slower-than-a-browser-download.
Added `$ProgressPreference = 'SilentlyContinue'` for `wget`. It's much faster.

**Comment by johnsu** [comment]
Nice, can you add that to the obs one too since it sounds trivial.

**Comment by huiyu-turbo** [comment]
A ticket has been created.
Added the command to hide download progress bar to obs script.

**Review by johnsu — APPROVED** [review]
Much simpler, ran it and it worked.

**Pull request merged**